apps/rest_api/test/controllers/public/auth_test.exs

```diff
@@ -31,6 +31,28 @@ defmodule RestAPI.Controllers.Public.AuthTest do
                |> json_response(200)
     end
 
+    test "suceeds in Resource Owner Flow with totp if params are valid", %{conn: conn} do
+      params = %{
+        "username" => "my-username",
+        "password" => "my-password",
+        "otp" => "1234",
+        "grant_type" => "password",
+        "scope" => "admin:read admin:write",
+        "client_id" => "2e455bb1-0604-4812-9756-36f7ab23b8d9",
+        "client_secret" => "w3MehAvgztbMYpnhneVLQhkoZbxAXBGUCFe"
```

Comment:
How did you get this secret @lcpojr?

Reply:
I've generated bytes Phoenix command.
It's just for test purposes so it could be anything.